README.md

```diff
@@ -4,139 +4,75 @@
   <img height="48" src="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
 </picture>
 
-# New Open Food Facts mobile app for Android and iPhone - Codename: "Smooth App"
+<br>
+
+# Smooth App : An app for Android and iPhone 
```

Comment:
This seems so normal, I recommend:

Smooth app, the new Open Food Facts mobile app for Android and iPhon